src/tests/index.test.js

```diff
@@ -0,0 +1,45 @@
+import commentsCounter from '../modules/commentsCounter.js';
+import pokemonsCount from '../modules/pokemonCount.js';
+
+describe('commentsCounter', () => {
+  beforeEach(() => {
+    const countText = document.createElement('span');
+    countText.id = 'comments-counter';
+    document.body.appendChild(countText);
+  });
+
+  afterEach(() => {
+    const countText = document.getElementById('comments-counter');
+    if (countText) {
+      countText.remove();
+    }
+  });
+
+  it('update comment counter successfully', async () => {
+    // Arrange
+    const count = 5;
+
+    // Act
+    await commentsCounter(count);
+
+    // Assert
+    const countText = document.getElementById('comments-counter');
+    expect(countText.innerText).toBe(`Comments (${count})`);
+  });
+});
+
```

Comment:
- [ ] Kindly consider writing tests for edge cases for both pokemonsCount and commentsCounter.
think about the edge case when there is 0 pokemon, so consider adding a test when pokemonsCount is 0.
and also consider adding same test for commentsCounter where there is 0 comments.